docs/customising/make_a_new_theme.md

```diff
@@ -79,7 +79,16 @@ setting in the config file.
 Set it to the URL of your uniquely named repo, for example:
 
     THEME_URLS:
-         - 'git://github.com/yourgithub/abcexample-theme.git'
+         - 'https://github.com/yourgithub/abcexample-theme.git'
+
+
+<div class="attention-box helpful-hint">
+  <p>
+    If you're familiar with git and already have ssh set up, feel free to use
+    the ssh format - <code>ssh://yourname@git.github.com/yourgithub/abcexample-theme.git</code>
```

Comment:
slight change of heart having looked at the rendered version, have expanded on that slightly to make the Github page more findable:

![screen shot 2016-04-04 at 10 59 02](https://cloud.githubusercontent.com/assets/27760/14244701/6a9240f0-fa54-11e5-83d9-ae4dc6a714a1.png)